dev/image/Dockerfile

```diff
@@ -139,7 +145,7 @@ RUN sudo install-packages \
     python-dev \
     python-setuptools
 
-RUN bash -c "pip uninstall crcmod; pip install --no-cache-dir -U crcmod"
+RUN sudo python3 -m pip uninstall crcmod; sudo python3 -m pip install --no-cache-dir -U crcmod
```

Comment:
Can you share a link to the build in werft that necessitated this change? We can begin dog fooding like this, but ideally this change wouldn't be necessary.

My thought is we should be able to swap like for like when going from the current full on docker hub to this new one. At least, if it's possible.

Reply:
One example https://werft.gitpod-dev.com/job/gitpod-build-prs-test-dazzle-rewrite.70

Reply:
Got it, thank you! 

The error was permissions related:
```shell
[dev/image:docker] Step 38/63 : RUN bash -c "python3 -m pip uninstall crcmod; python3 -m pip install --no-cache-dir -U crcmod"
[dev/image:docker]  ---> Running in 6f7a4af96876
[dev/image:docker] [91mWARNING: Skipping crcmod as it is not installed.
[dev/image:docker] [0mCollecting crcmod
[dev/image:docker]   Downloading crcmod-1.7.tar.gz (89 kB)
[dev/image:docker] Building wheels for collected packages: crcmod
[dev/image:docker]   Building wheel for crcmod (setup.py): started
[dev/image:docker]   Building wheel for crcmod (setup.py): finished with status 'done'
[dev/image:docker]   Created wheel for crcmod: filename=crcmod-1.7-py3-none-any.whl size=18840 sha256=580e9c7e56175ac26f60175c7b1592cdbac0007696934f9cc4b5189b1fe9fdd7
[dev/image:docker]   Stored in directory: /tmp/pip-ephem-wheel-cache-zlxkbes3/wheels/ca/5a/02/f3acf982a026f3319fb3e798a8dca2d48fafee7761788562e9
[dev/image:docker] Successfully built crcmod
[dev/image:docker] Installing collected packages: crcmod
[dev/image:docker] [91mERROR: Could not install packages due to an EnvironmentError: [Errno 13] Permission denied: '/workspace/.pip-modules'
[dev/image:docker] Check the permissions.
```

Let's plan on solving in a follow-up issue, while the org is dog fooding.